src/Apps/ArtQuiz/ArtQuizInterface/index.ts

```diff
@@ -0,0 +1 @@
+export * from "./ArtQuizInterface"
```

Comment:
i'll say it till the end of my days... no need for arbitrary index files 😄

Reply:
I love them. They make imports so much neater. 😭

Reply:
In the right spot they can certainly work great, I agree; we should use them sparingly though -- for example where there are a lot of shared exports [such as in palette](https://github.com/artsy/palette/blob/main/packages/palette/src/index.tsx) or where there's lib-esque code that will in the end have a lot of exports (this likely wont)